Source/Core/VideoBackends/Vulkan/VulkanContext.cpp

```diff
@@ -223,20 +286,21 @@ bool VulkanContext::SelectInstanceExtensions(std::vector<const char*>* extension
   }
 #endif
 
-  // VK_EXT_debug_report
-  if (enable_debug_report && !AddExtension(VK_EXT_DEBUG_REPORT_EXTENSION_NAME, false))
-    WARN_LOG_FMT(VIDEO, "Vulkan: Debug report requested, but extension is not available.");
-
   AddExtension(VK_KHR_GET_PHYSICAL_DEVICE_PROPERTIES_2_EXTENSION_NAME, false);
   if (wstype != WindowSystemType::Headless)
   {
     AddExtension(VK_KHR_GET_SURFACE_CAPABILITIES_2_EXTENSION_NAME, false);
   }
 
+  // VK_EXT_debug_utils
   if (AddExtension(VK_EXT_DEBUG_UTILS_EXTENSION_NAME, false))
   {
     g_Config.backend_info.bSupportsSettingObjectNames = true;
   }
+  else
+  {
+    WARN_LOG_FMT(VIDEO, "Vulkan: Debug utils requested, but extension is not available.");
+  }
```

Comment:
Does the log message here still make sense? Before "requested" corresponded to `enable_debug_report`, but now there doesn't seem to be any similar check.

Reply:
Changed it to only emit the warning if `enable_debug_utils` is true.